ETWProviders/etwprof.cpp

```diff
@@ -120,28 +120,35 @@ class CETWRegister
 } g_ETWRegister;
 
 // Redirector function for EventRegister. Called by macros in ETWProviderGenerated.h
-ULONG EVNTAPI EventRegister( LPCGUID ProviderId, PENABLECALLBACK EnableCallback, PVOID CallbackContext, PREGHANDLE RegHandle )
+#pragma warning(suppress: 28253)

```

Comment:
Good point......I was annoyed enough with the fact that Microsoft didn't properly annotate their own function - moreso than usual because we are redirecting - that I wasn't thinking. I should fix this tomorrow?